samples/sample-user-camera-dropdown-plugin/src/sample-user-camera-dropdown-plugin-item/component.tsx

```diff
@@ -2,26 +2,114 @@ import * as React from 'react';
+
+  const userCamera = pluginApi.useUserCameraDomElements(
+    videoStreams?.user_camera.map((vs) => vs.streamId),
+  );
+
+  pluginLogger.info(`logging the domElements manipulation for userCamera: (${userCamera}) for streams (${videoStreams})`);
+
   useEffect(() => {
+    const buttonScreenshare1 = new ScreenshareHelperButton({
+      icon: 'user',
+      disabled: false,
+      label: 'This will log on the console',
+      tooltip: 'this is a button injected by plugin',
+      position: ScreenshareHelperItemPosition.TOP_RIGHT,
+      onClick: () => {
+        pluginLogger.info('Log from nav bar plugin');
+      },
+      hasSeparator: true,
+    });
+
+    const buttonScreenshare5 = new ScreenshareHelperButton({
+      icon: 'popout_window',
+      disabled: false,
+      label: 'This will log on the console',
+      tooltip: 'this is a button injected by plugin',
+      position: ScreenshareHelperItemPosition.TOP_RIGHT,
+      onClick: () => {
+        pluginLogger.info('Log from nav bar plugin');
+      },
+      hasSeparator: true,
+    });
+
+    const buttonScreenshare2 = new ScreenshareHelperButton({
+      icon: 'undo',
+      disabled: false,
+      label: 'This will log on the console',
+      tooltip: 'this is a button injected by plugin',
+      position: ScreenshareHelperItemPosition.TOP_LEFT,
+      onClick: () => {
+        pluginLogger.info('Log from nav bar plugin');
+      },
+      hasSeparator: true,
+    });
+
+    const buttonScreenshare3 = new ScreenshareHelperButton({
+      icon: 'settings',
+      disabled: false,
+      label: 'This will log on the console',
+      tooltip: 'this is a button injected by plugin',
+      position: ScreenshareHelperItemPosition.BOTTOM_LEFT,
+      onClick: () => {
+        pluginLogger.info('Log from nav bar plugin');
+      },
+      hasSeparator: true,
+    });
+
+    const buttonScreenshare4 = new ScreenshareHelperButton({
+      icon: 'plus',
+      disabled: false,
+      label: 'This will log on the console',
+      tooltip: 'this is a button injected by plugin',
+      position: ScreenshareHelperItemPosition.BOTTOM_RIGHT,
+      onClick: () => {
+        pluginLogger.info('Log from nav bar plugin');
+      },
+      hasSeparator: true,
+    });
+    pluginApi.setScreenshareHelperItems([buttonScreenshare1, buttonScreenshare5,
+      buttonScreenshare2, buttonScreenshare3, buttonScreenshare4,
+    ]);
+  }, []);
+
+  useEffect(() => {
+    const randomElement = videoStreams?.user_camera[
+      Math.floor(Math.random() * (videoStreams?.user_camera.length ?? 0))
+    ];
+
     pluginApi.setUserCameraDropdownItems([
       new UserCameraDropdownSeparator(),
       new UserCameraDropdownOption({
         label: 'This will log on the console',
         icon: 'user',
-        onClick: () => {
-          pluginLogger.info('Alert sent from plugin');
+        displayFunction: ({ userId }) => randomElement?.user.userId === userId,
+        onClick: ({ userId }) => {
```

Comment:
What if the user have 2 webcams?
I believe we need more parameters here.
BTW, I would add:
- userId
- streamId
- videoDomElement

So the dom element would be available without the need to call another hook.